packages/backend/src/api/controllers/getEscapableAssets.ts

```diff
@@ -0,0 +1,79 @@
+import { EscapableAssetEntry } from '@explorer/frontend'
+import { CollateralAsset, PageContext } from '@explorer/shared'
+import { StarkKey } from '@explorer/types'
+import uniqBy from 'lodash/uniqBy'
+
+import { UserTransactionRepository } from '../../peripherals/database/transactions/UserTransactionRepository'
+import { WithdrawableAssetRepository } from '../../peripherals/database/WithdrawableAssetRepository'
+import { getCollateralAssetDetails } from './getCollateralAssetDetails'
+
+export async function getEscapableAssets(
```

Comment:
I'm not sure if separate file is a good place, maybe we can create `WithdrawableAssetService` and put there the stark key/eth address logic behind withdrawable asset and this function?